roxie/ccd/ccdserver.cpp

```diff
@@ -12247,7 +12247,14 @@ class CRoxieServerIndexWriteActivity : public CRoxieServerInternalSinkActivity,
         if(clusterHandler)
             clusterHandler->splitPhysicalFilename(dir, base);
         else
-            splitFilename(filename.str(), &dir, &dir, &base, &base);
+        {
+            // Check filename is URL and get localpath, only actually necessary if force remote reads are are on
```

Comment:
This is a disk write - is force remote reads supposed to also affect them?

Reply:
yes, there's no distinction. You specify a wildcard path match and all files that match are redirected.

Reply:
In that case the function enableForceRemoteReads is poorly named...